cumulusci/tasks/release_notes/task.py

```diff
@@ -13,8 +12,8 @@ class GithubReleaseNotes(BaseGithubTask):
             'required': True,
         },
         'publish': {
```

Comment:
I wouldn't change it here, but we have a task with `dry run` (default True) and a task with `publish` (default False) and i wonder if we should try to be consistent if thats a positive or a negative.

i could see that becoming confusing if we wanted to be able to run a release flow that does everything dry first.